option optimize_for = SPEED;
option java_package = "routing";

import "common.proto";
import "clientMessage.proto";

message CommandMessage {
   optional Header header = 1;

    oneof payload {
  
      bool ping = 3;

      string message = 4;
      
      Failure err = 5;
      
      RequestMessage reqMsg = 6;
      
      ResponseMessage resMsg =7;
   }
}
